<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncryptionSettingsWindow</class>
 <widget class="QWidget" name="EncryptionSettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBoxShowNamesAsHashes">
     <property name="text">
      <string>Show only hashed names for user</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxHashStoredData">
     <property name="text">
      <string>Hash stored data</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxEncryptStoredData">
     <property name="text">
      <string>Encrypt stored data</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonDeleteStoredData">
     <property name="text">
      <string>Delete locally stored data</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Changes will be
applied after restart
of the application</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
